internal/storages/s3/config.go

```diff
@@ -50,6 +51,9 @@ func NewConfig() *Config {
 		ForcePathStyle: true,
 		MaxRetries:     defaultMaxRetries,
 		MaxPartSize:    defaultMaxPartSize,
+		Bucket:   		os.Getenv("STORAGE_S3_BUCKET_NAME"),
+		Region: 		os.Getenv("STORAGE_S3_BUCKET_REGION"),
+		Prefix:			os.Getenv("STORAGE_S3_BUCKET_PREFIX"),
 	}
```

Comment:
Hi! Greenmask employs Viper and Cobra frameworks for managing the configuration. This means you should not hardcode `os.Getenv`. 

Have a look at [an example](https://github.com/GreenmaskIO/greenmask/blob/main/cmd/greenmask/cmd/root.go#L114)

Reply:
@wwoytenko I see, while investigating the code further I realized that Viper would take care of the configuration. As I understand, I believe that it was supposed to load all the configuration from environment variables automatically since   `viper.AutomaticEnv()` is being called in the `initConfiguration()` function [here](https://github.com/GreenmaskIO/greenmask/blob/cd3d273dda3b906ce75079be5bbafdcb933e7e4d/cmd/greenmask/cmd/root.go#L143C2-L143C22).

However, I can't seem to make it work using any environment variables following Viper's docummentation (like `STORAGE_DIRECTORY_PATH` or `COMMON_TMP_DIR`). I'm still investigating but I'll surely drop those modifications.

Reply:
@wwoytenko it seems I was able to fix it, it was simply the lack of the separator replacer. I opened a new PR #43 with the change.

On a side note I would like to propose a couple of other changes (and I'm unsure where would be the ideal location to communicate those, so I'm just adding them here):
- make `config.yaml` not required -- this would make it easier to run greenmask as a container without needing to rebuild the image with a config file or mounting a volume with the file
- set `common.tmp_dir` default value to linux `/tmp`
- change `Dockerfile` entrypoint to `greenmask` instead of `/bin/bash` -- this is more of a matter of following convention of other dockerized tools
- define a user `greenmask` on the container to avoid running it as root -- this is a security best practice I believe

I can work on all of those if you agree, I've tested a few of those changes locally already and they are really simple, the first one being the most impactful for my use case. To give you some context of my use case: I'm planning to run greenmask once a day as a ECS container inside a VPC in AWS, export dumps to S3 and also use it in Github actions to restore the dump to a postgres database so we can run database migration tests in our CI. So, having all the configurations as environment variables would be ideal, even the transformer configurations for the dump operation (which I still haven't tested yet).

Reply:
@joao-zanutto sure you can start doing it with all the listed fixes/features. Kindly remind you to review the documentation according to the changes will make. 

Thank you!